content/plugins/aws.md

```diff
@@ -152,10 +190,182 @@ See the [common Runtime Properties section](#runtime-properties).
+
+**Attributes:**
+
+See the [common Runtime Properties section](#runtime-properties).
+
+Note that the customer gateway's id in AWS is available via the `aws_resource_id` runtime-property.
+
+
+## cloudify.aws.nodes.ACL
+
+**Derived From:** [cloudify.nodes.Root](reference-types.html)
+
+**Properties:**
+
+  * `acl_network_entries` A list of of network acl entries. See the `cloudify.datatypes.aws.NetworkAclEntry` in the plugin.yaml data definitions section for its structure.
+
+**Mapped Operations:**
+
+  * `cloudify.interfaces.lifecycle.create` creates a network ACL
+  * `cloudify.interfaces.lifecycle.delete` deletes a network ACL
+  * `cloudify.interfaces.validation.creation` see [common validations section](#Validations).
+
+**Attributes:**
+
+See the [common Runtime Properties section](#runtime-properties).
+
+Note that the network_acl's id in AWS is available via the `aws_resource_id` runtime-property.
+
+
+## cloudify.aws.nodes.DHCPOptions
+
+**Derived From:** [cloudify.nodes.Root](reference-types.html)
+
+**Properties:**
+
+  * `domain_name` The domain name that you associate with your DCHP.
+  * `domain_name_servers` A list of existing Domain Name Servers' IP addresses.
+  * `ntp_servers` A list of existing NTP Servers' IP addresses.
+  * `netbios_name_servers` A list of existing NetBIOS Servers' IP addresses.
+  * `netbios_node_type` The type of Net BIOS.
+
+**Mapped Operations:**
+
+  * `cloudify.interfaces.lifecycle.create` creates a DHCP Options Set
+  * `cloudify.interfaces.lifecycle.delete` deletes a DHCP Options Set
+  * `cloudify.interfaces.validation.creation` see [common validations section](#Validations).
+
+**Attributes:**
+
+See the [common Runtime Properties section](#runtime-properties).
+
+Note that the dhcp_option_set's id in AWS is available via the `aws_resource_id` runtime-property.
+
+
+## cloudify.aws.nodes.RouteTable
+
+**Derived From:** [cloudify.nodes.Root](reference-types.html)
+
+**Mapped Operations:**
+
+  * `cloudify.interfaces.lifecycle.create` creates a Route Table
+  * `cloudify.interfaces.lifecycle.delete` deletes a Route Table
+  * `cloudify.interfaces.validation.creation` see [common validations section](#Validations).
+
+**Attributes:**
+
+See the [common Runtime Properties section](#runtime-properties).
+
+Note that the route_table's id in AWS is available via the `aws_resource_id` runtime-property.
+
 
 # Relationships
 
-A [relationship](reference-terminology.html#relationship) represents a dependency and/or a connection between [nodes](reference-terminology.html#node) in Cloudify. For example, an instance can be created within a Security Group. Or an Instance can depend on a keypair.
+A [relationship](reference-terminology.html#relationship) represents a dependency and/or a connection between [nodes](reference-terminology.html#node) in Cloudify. For example, an instance can be created within a Security Group.
```

Comment:
I'm not sure why you are explaining here about relationships - this is a reference doc. It's better to remove it.